app/Http/Controllers/Api/CustomFieldsController.php

```diff
@@ -26,9 +26,7 @@ public function index()
     {
         $this->authorize('index', CustomFields::class);
         $fields = CustomField::get();
-
-        $total = count($fields);
-        return (new CustomFieldsTransformer)->transformCustomFields($fields, $total);
+        return (new CustomFieldsTransformer)->transformCustomFields($fields, $fields->count());
```

Comment:
Testing linking back from discourse via commit comment. Ignore this :)
http://discourse.snipeitapp.com/t/this-is-a-test-post-in-bug-reports/17/2